plugins/org.jkiss.dbeaver.model/src/org/jkiss/dbeaver/runtime/ui/DBPPlatformUI.java

```diff
@@ -136,6 +136,11 @@ DBPAuthInfo promptUserCredentials(
      */
     String promptProperty(String prompt, String defValue);
 
+    /**
+     * Ask user to enter a multiline value
+     */
+    String promptText(String title, String prompt, String defValue);
```

Comment:
Although no surrounding methods are annotated with `@NotNull`/`@Nullable` annotations, it's pretty much always worth doing so in new APIs.